tools/extra/pac/fpgabist/bist_common.py

```diff
@@ -82,12 +81,14 @@ def get_afu_id(gbs_path="", bdf=None):
 
 # Return a list of all available bus numbers
 def get_all_fpga_bdfs(args):
-    bdf_pattern = re.compile(BDF_PATTERN)
+    pattern = re.compile(r'\d+:(?P<bus>[a-fA-F0-9]{2}):'
```

Comment:
Doesn't this use the original pattern you removed above?

Reply:
Yes.

Reply:
Yes.